occrp/templates/story.html

```diff
@@ -17,13 +17,21 @@
 
 <div class="row">
     <div class="col-md-12">
-    {% include 'partials/event_form.html' %}
+        {% include 'partials/event_form.html' %}
     </div>
 </div>
 
 {% if story.events %}
 <div class="row">
-    <div class="col-md-12">
+    <div class="col-md-4">
+        <!-- Facet panel -->
+        <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+            {% for facet in facets %}
+                {% include 'partials/facets.html' %}        
+            {% endfor %}
+        </div>
+    </div>
+    <div class="col-md-8">
```

Comment:
You might consider putting the "Events" header outside the 2 columns so that it sits on top of both of them. Maybe you already tried this and it looked stupid but anyways, when I was looking at it, it seemed like there was a bunch of padding on top of the word "Events" that made the parts look separate.

![screenshot-2017-8-8 occrp](https://user-images.githubusercontent.com/551491/29075555-5a800dfe-7c18-11e7-8fd3-58c67196d802.png)